sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -138,7 +145,7 @@ Other noteworthy changes:
 - [Receiving events with checkpointing](#migrating-code-from-eventprocessorhost-to-eventhubconsumerclient-for-receiving-events)
 - [Sending events](#migrating-code-from-eventhubclient-to-eventhubproducerclient-for-sending-events)
 
-### Migrating code from `EventHubClient` to `EventHubConsumerClient` for receiving events
+### Migrating from `EventHubClient` to `EventHubConsumerClient` for receiving events
```

Comment:
In the v2 code snippet here we show how to pass the event position, but not in the v5 code snippet

Reply:
Thank you, fixed.